lib/xmpp.php

```diff
@@ -20,14 +20,16 @@ class OC_User_XMPP extends \OCA\user_external\Base {
 	private $xmppDbUser;
 	private $xmppDbPassword;
 	private $xmppDomain;
+	private $passwordHashed;
 
-	public function __construct($host, $xmppDb, $xmppDbUser, $xmppDbPassword, $xmppDomain) {
+	public function __construct($host, $xmppDb, $xmppDbUser, $xmppDbPassword, $xmppDomain, $passwordHashed) {
```

Comment:
I guess we need to set a default value here: `$passwordHashed = true` to not crash existing setups if the configuration is not updated...? Otherwise this will raise an exception after updating, if the config.php is not changed...
Or do you have a set a default somewhere else?

Reply:
Implemented.